pkg/meta/tkv_tikv.go

```diff
@@ -172,6 +172,19 @@ func (c *tikvClient) name() string {
 	return "TiKV"
 }
 
+func (c *tikvClient) isEmpty() bool {
+	tx, err := c.client.Begin()
+	if err != nil {
+		panic(err)
+	}
+	it, err := tx.Iter(nil, nil)
```

Comment:
Check with the prefix, we could use the tx.exists()

Reply:
Prefix is optional

Reply:
Nvm, we can also modify `nextKey`

Reply:
We should give a warning if prefix is empty

Reply:
0xFF could be used when prefix is empty

Reply:
Isn't that reserved for internal keys?